requirements.txt

```diff
@@ -0,0 +1,26 @@
+# NVDA's build system is SCons
+SCons==4.1.0.post1
+
+# NVDA's runtime dependencies
+comtypes==1.1.8
+pyserial==3.5
+wxPython==4.1.1

```

Comment:
Installing wx like that will also pull NumPy from PyPI. As mentioned by @josephsl in #11967 if NumPy is installed Py2exe bundles parts of in in NVDA's binary builds because ComTypes has an optional dependency on NumPy arrays. NumPy is also a dependency of screencaplibrary. Perhaps we need to be able to specify `--no-deps` for a particular package?

Reply:
NumPy will be excluded from the Py2exe bundle explicitly, see `source/setup.py`. I think NumPy is required indirectly by robotframework-screencaplibrary anyway.